GoogleForCreators/web-stories-wp
# Add flow to create a post after publishing a story

## Summary

Adds a new dialog that appears after publishing a story

## Relevant Technical Choices

For accessibility reasons, updates button components to make it possible to create a link that looks like a button. Buttons were missing theme styling, so I added some to make sure the button-styled links look reasonable.

## To-do

* [x] Add E2E Tests for the full flow

## User-facing changes

New post-publish dialog:

<img width="1637" alt="Screenshot 2020-06-05 at 11 27 54" src="https://user-images.githubusercontent.com/841956/83863381-8ed15980-a723-11ea-9243-847d81c645bd.png">

New post pre-filled with the embedded story:

<img width="1678" alt="Screenshot 2020-06-05 at 11 50 13" src="https://user-images.githubusercontent.com/841956/83863564-c6d89c80-a723-11ea-964a-1da23cc79e21.png">


## Testing Instructions

1. Create a new, stunning story
1. Publish the story
1. Verify that a dialog appears
1. Click on "Add to new post" and see how the story is embedded within the post.

For QA:

- Repeat once for a story with and without a poster image (aka featured image)
- Repeat and verify that "Dismiss" closes the dialog
- Verify that the dialog does not appear again when making further edits to the published story and clicking "Update"
- Verify that the dialog does not appear when first scheduling a story to be published in the future and then changing back to publish immediately.


---



Fixes #1118
